OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory61.cs

```diff
@@ -214,6 +215,9 @@ private unsafe struct CombatantMemory
             [FieldOffset(0x1AB2)]
             public ushort WorldID;
 
+            [FieldOffset(0x1AC6)]
+            public byte WeaponId;
```

Comment:
Belatedly, inconsistent "Id" capitalization here with the rest of the file.  Not sure what to do about that.  Maybe we can send both for a little bit?